src/main/java/com/weather/model/Weather.java

```diff
@@ -1,7 +1,17 @@
 package com.weather.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public class Weather extends WeatherEntry{
```

Comment:
Entity class have different purpose of use than the Model class. So they can't be extends Model classes.

Reply:
Yes I will make make different entity and model classes.